docs/en_US/sqls/built-in_functions.md

```diff
@@ -179,6 +179,21 @@ When casting to datetime type, the supported column type and casting rule are:
 
 Currently, 'zlib', 'gzip', 'flate' and 'zstd' method are supported.
 
+## Array Functions
+
+| Function   | Example                   | Description                                                           |
+|------------|---------------------------|-----------------------------------------------------------------------|
+| array_position | array_postion(array, value) | Return a 1-based index of the first occurrence of val if it is found within array. If val does not exist within array, it returns 0 |
```

Comment:
1-based, is it better to be 0-based?

Reply:
Both are ok to me, for now rockset used 1-based.

Reply:
I prefer 0-based because currently all our indexes are 0-based. It is weird to have 1 based here.

Reply:
updated